openslides/mediafiles/serializers.py

```diff
@@ -1,14 +1,38 @@
-from openslides.utils.rest_api import ModelSerializer, SerializerMethodField
+import mimetypes
 
+from django.db import models as dbmodels
+from rest_framework.fields import FileField
+
+from ..utils.rest_api import ModelSerializer, SerializerMethodField
```

Comment:
@ostcar I think this style `..` is better than ours but do `from openslides.utils...` all the time. What do you think?